Gopkg.toml

```diff
@@ -64,3 +64,17 @@ required = [
 [[constraint]]
   name = "k8s.io/code-generator"
   version = "kubernetes-1.9.1"
+
+[prune]
+  non-go = true
+  go-tests = true
+  unused-packages = true
```

Comment:
My understanding is that this says,

Do not include:
- non-go files
- go tests
- pkgs we don't build

Reply:
Yes, with the exception of repo-infra because some of the code-gen related files are .txt files, etc and we don't want those nuked.